plugin/src/main/java/dev/regadas/trino/pubsub/listener/Encoder.java

```diff
@@ -31,7 +31,10 @@ public static Encoding fromOrDefault(String encoding, Encoding defaultEncoding)
     @FunctionalInterface
     interface MessageEncoder extends Encoder<Message> {
         JsonFormat.Printer JSON_PRINTER =
-                JsonFormat.printer().preservingProtoFieldNames().omittingInsignificantWhitespace();
+                JsonFormat.printer()
+                        .preservingProtoFieldNames()
+                        .omittingInsignificantWhitespace()
+                        .includingDefaultValueFields();
```

Comment:
Just double checking here ...do we really need this? I'm thinking that there are cases where i.e having 0 means different from absence (null) and if we have default values for everything might send us in the wrong direction.

Reply:
Hmm now that I consider it, i think you're right... since this would be the JSON on the output stage from the plugin into PubSub, and not rather the serialization of the internal fields into json.  My mistake!